estuary/values.yaml

```diff
@@ -0,0 +1,17 @@
+# make a dependency when kdp is published
+presto:
+  url: "http://kdp-kubernetes-data-platform-presto.kdp:8080"
+
+image:
+  repository: smartcitiesdata/apps/estuary
+  tag: development
+  pullPolicy: IfNotPresent
+
+# defaults to be overridden by user
+replicaCount: 1
```

Comment:
Do we want to use more than one replica in case estuary goes down?

Reply:
I don't think this app is so much critical at this point, so it will unnecessary add extra cost but we may increase it at anytime.

Reply:
I don't think this app is so much critical at this point, so it will unnecessary add extra cost but we may increase it at anytime.

Reply:
Let's talk about this as a group. It sounds like a business value question and we as devs should be identifying these sorts of questions, gathering data, and then presenting options to the PO.

Reply:
I think since kafka keeps track of the offset and we are persisting the events read we should need an extra replica. If we are up to date with the latest events it shouldn't be the end of the world, because we will get the events once estuary gets stood up again.